### DISCUSSION TITLE
get help to fork a copy

### DISCUSSION THREAD

**Discussion by mrchanshares** [File-New-Project/EarTrumpet#1106] [Q&A] [answered]
I want to develop a fork but get following errors after cloning a copy to visual studio 2022. How can I fix that? Thanks

1>------ Build started: Project: EarTrumpet, Configuration: Debug|x86 ------
2>------ Build started: Project: EarTrumpet.Package, Configuration: Debug|x86 ------
1>C:\Users\lib20\Source\Repos\EarTrumpet\EarTrumpet\EarTrumpet.csproj(496,30): error MSB4057: The target "GetVersion" does not exist in the project.
3>------ Build started: Project: EarTrumpet.ColorTool, Configuration: Debug|x86 ------
3>C:\Users\lib20\Source\Repos\EarTrumpet\EarTrumpet.ColorTool\MainWindow.xaml(18,18): error MC3074: The tag 'Binding' does not exist in XML namespace 'clr-namespace:EarTrumpet.Extensions.EventBinding;assembly=EarTrumpet'. Line 18 Position 18.
2>C:\Users\lib20\Source\Repos\EarTrumpet\EarTrumpet\EarTrumpet.csproj(492,5): error : This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is ..\packages\GitVersionTask.5.5.1\build\GitVersionTask.props.
========== Build: 0 succeeded, 3 failed, 0 up-to-date, 0 skipped ==========
Build failed.

**Comment by riverar** [comment, marked as answer]
We have compilation instructions at https://github.com/File-New-Project/EarTrumpet/blob/master/COMPILING.md.

**Comment by mrchanshares** [comment]
I am using Window 11. So should all reference to be changed to Window 11? For example, [Windows 10 SDK (10.0.14393.0)].

**Reply by riverar** [reply]
No, because the app targets Windows 10 and Windows 11.

**Reply by mrchanshares** [reply]
Can you also advise where is the source code of the entry point to the volume mixer UI window?

**Comment by mrchanshares** [comment]
I probably will redo all of them and start fresh again tomorrow. Thanks.

**Comment by mrchanshares** [comment]
I have successfully build the solution using visual studio 2017 on window 10.
But the debugging is being shutdown after executing  SingleInstanceAppMutex.TakeExclusivity().
Does that need a fix?

I have commented out the shutdown() line and bypassed it. Now, I can run to DisplayFirstRunExperience();
